<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>RegisterWindow</class>
  <widget class="QWidget" name="RegisterWindow">
	<property name="geometry">
	  <rect>
		<x>0</x>
		<y>0</y>
		<width>400</width>
		<height>360</height>
	  </rect>
	</property>
	<property name="minimumSize">
	  <size>
		<width>400</width>
		<height>360</height>
	  </size>
	</property>
	<property name="windowTitle">
	  <string>创建账号</string>
	</property>
	<property name="styleSheet">
	  <string notr="true">
		QWidget { background-color: #111111; color: white; }
		QLineEdit { background-color: #1e1e1e; border: 1px solid #444444; border-radius: 4px; padding: 6px; }
		QPushButton { background-color: #7b3ff2; color: white; border-radius: 6px; padding: 8px; font-weight: 600; }
		QPushButton:disabled { background-color: #555555; }
		QLabel.link { color: #a26bff; }
	  </string>
	</property>

	<layout class="QVBoxLayout" name="mainLayout">
	  <property name="leftMargin">
		<number>40</number>
	  </property>
	  <property name="topMargin">
		<number>40</number>
	  </property>
	  <property name="rightMargin">
		<number>40</number>
	  </property>
	  <property name="bottomMargin">
		<number>40</number>
	  </property>
	  <property name="spacing">
		<number>16</number>
	  </property>

	  <!-- 标题 “创建账号” -->
	  <item>
		<widget class="QLabel" name="titleLabel">
		  <property name="text">
			<string>创建账号</string>
		  </property>
		  <property name="alignment">
			<set>Qt::AlignCenter</set>
		  </property>
		  <property name="font">
			<font>
			  <pointsize>18</pointsize>
			  <bold>true</bold>
			  <weight>75</weight>
			</font>
		  </property>
		</widget>
	  </item>

	  <!-- 描述文字 -->
	  <item>
		<widget class="QLabel" name="descLabel">
		  <property name="text">
			<string>创建账号以同步你的笔记</string>
		  </property>
		  <property name="alignment">
			<set>Qt::AlignCenter</set>
		  </property>
		  <property name="wordWrap">
			<bool>true</bool>
		  </property>
		</widget>
	  </item>

	  <!-- 用户名 -->
	  <item>
		<widget class="QLabel" name="usernameLabel">
		  <property name="text">
			<string>用户名</string>
		  </property>
		</widget>
	  </item>
	  <item>
		<widget class="QLineEdit" name="usernameEdit">
		  <property name="placeholderText">
			<string></string>
		  </property>
		</widget>
	  </item>

	  <!-- 密码 -->
	  <item>
		<widget class="QLabel" name="passwordLabel">
		  <property name="text">
			<string>密码</string>
		  </property>
		</widget>
	  </item>
	  <item>
		<widget class="QLineEdit" name="passwordEdit">
		  <property name="echoMode">
			<enum>QLineEdit::Password</enum>
		  </property>
		</widget>
	  </item>

	  <!-- 提示信息 -->
	  <item>
		<widget class="QLabel" name="messageLabel">
		  <property name="text">
			<string></string>
		  </property>
		</widget>
	  </item>

	  <!-- 注册按钮 -->
	  <item>
		<widget class="QPushButton" name="registerButton">
		  <property name="text">
			<string>注册</string>
		  </property>
		</widget>
	  </item>

	  <!-- 伸缩，相当于 addStretch() -->
	  <item>
		<spacer name="verticalSpacerBottom">
		  <property name="orientation">
			<enum>Qt::Vertical</enum>
		  </property>
		  <property name="sizeHint">
			<size>
			  <width>20</width>
			  <height>40</height>
			</size>
		  </property>
		</spacer>
	  </item>

	  <!-- 底部 “已经有账号了？ 登录” -->
	  <item>
		<layout class="QHBoxLayout" name="bottomLayout">
		  <property name="spacing">
			<number>4</number>
		  </property>

		  <!-- 左侧伸缩 -->
		  <item>
			<spacer name="horizontalSpacerLeft">
			  <property name="orientation">
				<enum>Qt::Horizontal</enum>
			  </property>
			  <property name="sizeHint">
				<size>
				  <width>40</width>
				  <height>20</height>
				</size>
			  </property>
			</spacer>
		  </item>

		  <!-- 提示文字 -->
		  <item>
			<widget class="QLabel" name="tipLabel">
			  <property name="text">
				<string>已经有账号了？</string>
			  </property>
			</widget>
		  </item>

		  <!-- 中间小间隔（相当于 addSpacing(4)） -->
		  <item>
			<spacer name="horizontalSpacerMiddle">
			  <property name="orientation">
				<enum>Qt::Horizontal</enum>
			  </property>
			  <property name="sizeHint">
				<size>
				  <width>4</width>
				  <height>20</height>
				</size>
			  </property>
			</spacer>
		  </item>

		  <!-- 登录标签 -->
		  <item>
			<widget class="QLabel" name="signInLabel">
			  <property name="text">
				<string>登录</string>
			  </property>
			  <property name="cursor">
				<cursorShape>PointingHandCursor</cursorShape>
			  </property>
			</widget>
		  </item>

		  <!-- 右侧伸缩 -->
		  <item>
			<spacer name="horizontalSpacerRight">
			  <property name="orientation">
				<enum>Qt::Horizontal</enum>
			  </property>
			  <property name="sizeHint">
				<size>
				  <width>40</width>
				  <height>20</height>
				</size>
			  </property>
			</spacer>
		  </item>
		</layout>
	  </item>
	</layout>
  </widget>
  <resources/>
  <connections/>
</ui>
